crates/kitsune_p2p/kitsune_p2p/src/gossip/simple_bloom/step_1_check_inner.rs

```diff
@@ -48,7 +48,7 @@ impl SimpleBloomMod {
         // TODO: clean up ugly locking here
         let needs_sync = self.inner.share_mut(|i, _| {
             Ok(i.initiate_tgt.is_none()
-                && i.last_initiate_check.elapsed().as_millis() as u32
+                && i.last_initiate_check.elapsed().unwrap().as_millis() as u32
```

Comment:
We try to never put `unwrap()`s in production code unless we can absolutely be sure that the result can't fail. In this case it can fail because `SystemTime` can move backwards and that's how elapsed can return an error.
This is why we use `Instant` as `Instant` is guaranteed to be monotonically growing.